pkg/metallb/metallb_test.go

```diff
@@ -366,6 +366,32 @@ func TestMetalLbWithOptions(t *testing.T) {
 	assert.Equal(t, "error", testBuilder.errorMsg)
 }
 
+func TestMetalLbWithFRRconfigAlwaysBlock(t *testing.T) {
```

Comment:
```suggestion
func TestMetalLbWithFRRConfigAlwaysBlock(t *testing.T) {
```
minor name change so it's camel cased

Reply:
Changed